src/pages/Models/Details/ModelDetails.js

```diff
@@ -145,9 +145,10 @@ const renderCounts = (activeView, modelStatusData) => {
   }
 
   return (
-    <Counts
-      primaryEntity={primaryEntity}
-      secondaryEntities={secondaryEntities}
+    <StatusStrip
+      statusList={{
+        [primaryEntity.label]: secondaryEntities,
```

Comment:
The relations page has a different UI to the rest of them, we should probably match it to the `primaryEntity.label` style.

Reply:
I don't know what you mean - can you clarify?